src/lib/dux/user/reducers.ts

```diff
@@ -0,0 +1,22 @@
+import { UserActionTypes, USER_ACTIONS } from './actionTypes';
+import initialState, { UserStoreStateType } from './initialState';
+
+export default function reducer(state: UserStoreStateType, action: UserActionTypes): UserStoreStateType {
+  switch (action.type) {
```

Comment:
ps. can I ask you to use ts-pattern if its not too much? https://github.com/sendbird/sendbird-uikit-react/blob/main/src/lib/dux/sdk/reducers.ts

Reply:
yea sure